gravatar/src/main/java/com/gravatar/AvatarUrl.kt

```diff
@@ -104,8 +104,10 @@ public class AvatarUrl {
     /**
      * Get the [URL] for the avatar.
      *
+     * @param cacheBuster Random string value to force a cache bust
```

Comment:
I wonder why Detekt hasn't complained about that... 🤔

Reply:
yeah good question